manuscript.Rmd

```diff
@@ -68,55 +68,154 @@ set.seed(42)
+A further complication arises through the fact that functionality of successful, modular scientific software is imported, re-used, and re-exported in other software packages.
+While we highly encourage this from both a software engineering perspective and for scientific progress, it poses further difficulties to citation-based metrics.
+This especially punishes highly relied-on backends such as optimization algorithms: For example, the NLopt optimization suite is cited 1,711 times on google scholar.
+However, the algorithms from that suite are used in countless software packages, for example, as a backend of OpenMx which again has 811 citations.
+Moreover, the algorithms from that suite are all open source, and therefore serve as a reference for others.
+As an example, consider the lme4 package, that adapted one of the algortihms from NLopt.
+This package has 57,820 citations on google scholar, yet no single lme4 user probably ever cited the NLopt library.
+We therefore argue that merrit statements are much more useful to assess impact of research software.
+
+## ID 11: GitHub Stars
+We strongly object to the metric Github stars.
+First of all, it is tied only to a single, commercial platform and allows no freedom to choose free alternatives such as local Gitlab instances or other platforms.
+More importantly, we question the validity of such metrics that like rating systems on commercial platforms are too easy to be gamed by creating fake accounts that provide high ratings.
+
+## ID 6: Date of most recent substantive update
+<!-- see issue #7 -->
+
+## ID 14: Lines of Code
+We discourage the lines of code (LOC) metric that was proposed to measure effort.
+There are several problems in using this indicator.
+First of all, lines of code for code contributions highly depends on the programming language, mastery and personal programming style.
+In particular, large LOC of a researcher may simply mean that they write inefficient and repetitive code, one of the great sins of programming.
+A feature of good software is modularity because it enables reuse of functions both inside and outside the project.
+A great example is the CRAN library that enables packages to built upon other packages and reuse their functionality. However, if LOC were a metric that a researcher are compared with, it encourages researchers to copy&paste chunks of code from other packages instead of importing them in their project.
+Second, lines of code is an ambiguous metric.
+It can be calculated differently depending on whether comments do count or whether physical lines of code or logical lines of code are counted.
+
+## ID 5: Date of first fully functional public version
+Date of first fully functional public version is a central field in the proposal as it serves as a denominator in computing citations relative to software age.
+In our own experience, research software often evolves very differently to the standards that software engineering prescribes.
```

Comment:
```suggestion
In our own experience, research software often evolves driven by time constraints and the need for specific functionality, not according to feature completeness.
```

Haha we argue for rigor and than say rigor is seldom achieved?

Reply:
well, this is why we need to encourage it. This is the whole point, right?!